### ISSUE TITLE
(Fabric) Crash with both Lithium and Artifacts present

### ISSUE THREAD

**Issue by InsaanNotFound** [TheIllusiveC4/Curios#127]
#### Versions:
* Curios: 0.0.13
* Fabric: 0.32.5 API / 0.11.3 Loader
* Trinkets: 2.6.7
* Artifacts: 3.2.0
* Lithium: 0.6.4
* Simple Backpacks: 1.0.36
* Things: 0.1
**Note:** Things and SB are used only to prove the issue, Trinkets is required lib for Artifacts, but does not cause crash on its own
#### Steps to Reproduce:
1. Obtain backpack from Simple Backpacks mod (inventory big enough to prove concept)
2. Get enough apples to fill all of 54 slots of the backpack
3. Apples have Curios support thanks to Things mod, yet this issue can be replicated with any stackable curios
4. Fill the backpack with apples at once
5. Close inventory with esc key
6. Game should crash, if it did not, it should after 2-3 times of doing the same.
...

Crash Log: https://pastebin.com/SW26VpMr

**Comment by InsaanNotFound** [comment]
Weirdly enough, game can crash on my heavily modded world with only 20 apples.
Things: https://www.curseforge.com/minecraft/mc-mods/things-fabric/files/3237801
Curios: https://www.curseforge.com/minecraft/mc-mods/curios-fabric/files/3233941
Trinkets: https://www.curseforge.com/minecraft/mc-mods/trinkets-fabric/files/3100959
Artifacts: https://www.curseforge.com/minecraft/mc-mods/artifacts-fabric/files/3269478
Lithium: https://www.curseforge.com/minecraft/mc-mods/lithium/files/3215441
Simple Backpack: https://www.curseforge.com/minecraft/mc-mods/simple-backpack-fabric/files/3172191
Fabric API: https://www.curseforge.com/minecraft/mc-mods/fabric-api/files/3248105
Fabric Loader: installed through MultiMC

Edit:
After some more testing if you are "lucky" enough, you can crash game with only Artifacts installed, without Lithium. But that takes alot more tries.

**Comment by TheIllusiveC4** [comment]
What happens if you don't have Artifacts installed?

**Comment by InsaanNotFound** [comment]
Game does not crash on trying this technique

**Comment by TheIllusiveC4** [comment]
Hmm, so it seems like Artifacts might be at the core of this issue. It could be a standalone issue with Artifacts or a specific cross-compatibility issue with Curios, not really sure. But I don't see anything that could cause this at a glance. This'll be a bit tricky to figure out.

You said this can be replicated with any stackable Curios. Which others have you tried? And have you tried containers other than the backpack?

**Comment by InsaanNotFound** [comment]
Issue can occur rarely with shulker boxes, but chances of that happening are very low. I modified Curious Necklace to be stackable; same effect

**Comment by TheIllusiveC4** [comment]
Shulker Boxes in what context? In the Curios inventory via Curious Shulker Boxes, or just inserting the items in regular Shulker Boxes placed in the world?

**Comment by InsaanNotFound** [comment]
As in inserting into regular shulker boxes; those would simply not drop; look up crashlog I sent in previous issue ticket about s.boxes

**Comment by InsaanNotFound** [comment]
Also I might have bad news...
I booted up my monstrous instance with 350 mods, but removed artifacts, created new world, filled a backpack full of apples, and everything was fine, then I tried to do the same with shulkerbox; it would not drop. Then I removed one stack of apples from backpack and inserted it again; game crashed. Noting that Artifacts were not there. It seems that the issue is rooted inside nature of Curios API.
